### PR TITLE
app doc update for 7 v2 apps - release 2

### PR DESCRIPTION
**Pull request by shivani-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/asana.md [hunk 1]
```diff
@@ -71,8 +71,12 @@ To set up [Cloud to Cloud Integration Asana Source](/docs/send-data/hosted-colle
 
 This section describes how to install the Asana app for Sumo Logic and description of the dashboard. You'll also need to configure the [Asana Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source/).
```
Comment (JV0812) [inline docs/integrations/saas-cloud/asana.md:72]:
```suggestion
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 2]
```diff
@@ -102,14 +102,14 @@ to_user, app_session_id  nodrop
...
 
 This section demonstrates how to install the Netskope App, and provides examples and descriptions for each of the dashboards. The Netskope App has the following components:
 
 * **Application Usage:** Insights into application usage; specifically by devices, users, users and traffic patterns.
 * **Security Alerts:** Visibility into Netskope security alerts and violations and the ability to identify effects of a breach.  
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Netskope Dashboards
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/netskope.md:114]:
```suggestion
## Viewing Netskope dashboards
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 3]
```diff
@@ -102,14 +102,14 @@ to_user, app_session_id  nodrop
 ```
 
 
-## Installing the Netskope App
+## Installing the Netskope app
 
 This section demonstrates how to install the Netskope App, and provides examples and descriptions for each of the dashboards. The Netskope App has the following components:
 
 * **Application Usage:** Insights into application usage; specifically by devices, users, users and traffic patterns.
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/netskope.md:109]:
```suggestion
* **Application Usage**. Insights into application usage; specifically by devices, users, users and traffic patterns.
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 4]
```diff
@@ -102,14 +102,14 @@ to_user, app_session_id  nodrop
...
 * **Security Alerts:** Visibility into Netskope security alerts and violations and the ability to identify effects of a breach.
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/netskope.md:110]:
```suggestion
* **Security Alerts**. Visibility into Netskope security alerts and violations and the ability to identify effects of a breach.  
```

File: docs/integrations/saas-cloud/asana.md [hunk 5]
```diff
@@ -69,10 +69,13 @@ To set up [Cloud to Cloud Integration Asana Source](/docs/send-data/hosted-colle
 
 ## Installing the Asana app
 
-This section describes how to install the Asana app for Sumo Logic and description of the dashboard. You'll also need to configure the [Asana Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source/).
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Asana dashboard
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/asana.md:75]: Although there's only one Asana dashboard, I'd still make this plural (_dashboards_) for consistency with its counterpart docs. We may add more in the future.

```suggestion
## Viewing Asana dashboards
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
1 change

**Review by kimsauce — APPROVED** [review]

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR https://github.com/SumoLogic/sumologic-documentation/pull/3433.

**Pull request closed without merging**